client/reader/stream/reader-tag-sidebar/index.jsx

```diff
@@ -16,24 +17,39 @@ import { recordReaderTracksEvent } from 'calypso/state/reader/analytics/actions'
 import { requestFollowTag, requestUnfollowTag } from 'calypso/state/reader/tags/items/actions';
 import { getReaderTagBySlug } from 'calypso/state/reader/tags/selectors';
 import { registerLastActionRequiresLogin } from 'calypso/state/reader-ui/actions';
+import getPrimarySiteId from 'calypso/state/selectors/get-primary-site-id';
 import '../style.scss';
 
 const ReaderTagSidebar = ( {
 	tag,
 	showFollow,
 	registerLastActionRequiresLogin: registerLastActionRequiresLoginProp,
 } ) => {
+	const primarySiteId = useSelector( ( state ) => getPrimarySiteId( state ) );
 	const translate = useTranslate();
 	const relatedMetaByTag = useRelatedMetaByTag( tag );
 	const tagStats = useTagStats( tag );
 	const dispatch = useDispatch();
 	const isFollowing = useSelector( ( state ) => getReaderTagBySlug( state, tag )?.isFollowing );
 	const isLoggedIn = useSelector( ( state ) => isUserLoggedIn( state ) );
+	const { data: prompts } = useBloggingPrompts( primarySiteId, 10 );
 
 	if ( relatedMetaByTag === undefined ) {
 		return null;
 	}
 
+	const showRecentPrompts = tag.indexOf( 'dailyprompt' === 0 ) || tag.indexOf( 'bloganuary' );
```

Comment:
There is a flaw in the logic here from what I can tell.. the `showRecentPrompts` is always true (I think) as I see the daily prompts when looking at `/tag/api` page.

This might work better?
```suggestion
	const showRecentPrompts = tag.startsWith('dailyprompt') || tag.includes('bloganuary');
```